### DISCUSSION TITLE
Stop z2m from removing devices?

### DISCUSSION THREAD

**Discussion by evlo** [Koenkk/zigbee2mqtt#15840] [Devices]
Is it possible for z2m to just keep paired devices as unavailable infinitely? Ie. if battery dies device "leaves the network" after few hours I think. Same if there is bug with the implementation of the device.

My goal is to see all devices I paired even if devices are offline or not seen on network for years.

**Comment by stigvi** [comment]
I cannot answer you, but your question is interesting. With zigbee christmas lights (Philips Hue Festavia), will it be "plug&play" after one year buried in a box in the cellar?
